src/foam/nanos/logger/DiscardLogger.java

```diff
@@ -0,0 +1,10 @@
+package foam.nanos.logger;
+
+public class DiscardLogger
+  extends ProxyLogger
+
+info(m) {
```

Comment:
So the java code is implemented in the js file, see foam/nanos/http/DefaultHttpParameters.js for an example.  Even though, at the moment, this is used entirely on the java side, we'll still add the   info(... method implementation for java in the js.